models/equflash/relaxation/optimizable.py

```
@@ -0,0 +1,716 @@
+            mask = torch.ones((3, 3), device=self.device)
+
+        # TODO make sure mask is on GPU
+        if mask.shape == (6,):
+            self.mask = torch.tensor(
+                voigt_6_to_full_3x3_stress(mask.detach().cpu()),
+                device=self.device,
+            )
+        elif mask.shape == (3, 3):
+            self.mask = mask
+        else:
+            raise ValueError("shape of mask should be (3,3) or (6,)")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)